src/tools/info/Makefile.am

```diff
@@ -18,6 +18,7 @@ build_config.h: $(top_builddir)/config.h Makefile
 
 ucx_info_CPPFLAGS  = $(BASE_CPPFLAGS)
 ucx_info_CFLAGS    = $(BASE_CFLAGS)
+ucx_info_LDFLAGS   = -lpthread $(BASE_CFLAGS)
```

Comment:
why pthread is needed?

Reply:
Because `ucx_info` includes headers that use libpthread. I don't know why does build work without ASAN (even if no `-lpthread` specified) and start failing  without this flag if ASAN flags are passed during the compilation.